docs/guides/site-to-site-connectivity/index.mdx

```diff
@@ -16,4 +16,4 @@ This section provides getting started guides for connecting to remote sites with
 | [APIs](apis.mdx)                | Connect to APIs at remote sites                                                        |
 | [APIs with mTLS](apis-mtls.mdx) | Connect to APIs at remote sites using end-to-end encryption with mTLS                  |
 | [DBs](dbs.mdx)                  | Connect to databases at remote sites                                                   |
-| [DBs with mTLS](apis.mdx)       | Connect to databases at remote sites using end-to-end encryption with mTLS and stunnel |
+| [DBs with mTLS](dbs-mtls.mdx)   | Connect to databases at remote sites using end-to-end encryption with mTLS and stunnel |
```

Comment:
is this just the wrong link? or did we change the name of this page?

Reply:
It was just the wrong link. Copy/pasta